esp-hal/src/spi/master.rs

```diff
@@ -1554,7 +1594,10 @@ mod dma {
         }
 
         /// Change the bus configuration.
-        // FIXME: when https://github.com/esp-rs/esp-hal/issues/2839 is resolved, add an appropriate `# Error` entry.
+        ///
+        /// # Errors.
+        /// If frequency passed in config exceeds 80Mhz, a corresponding
```

Comment:
we should tell the real limit (which is different for H2) - we already do it in other places